app/controllers/reports_controller.rb

```diff
@@ -142,7 +142,7 @@ def pick_entity()
       if @entity == 'Patient'
         @list = Patient.where(patient_id: params[:query])
         if @list.empty?
-          @list = Patient.where("name LIKE :prefix", prefix: "%#{params[:query]}%")
+          @list = Patient.where("first_name LIKE :prefix or last_name LIKE :prefix", prefix: "%#{params[:query]}%")
```

Comment:
Also, you can use `ILIKE` for a case-insensitive match.

On Wed, Jun 24, 2015 at 12:36 PM, Albert J wong <notifications@github.com>
wrote:

> In app/controllers/reports_controller.rb
> <https://github.com/awong-dev/SCIInterim/pull/114#discussion_r33186580>:
>
> > @@ -142,7 +142,7 @@ def pick_entity()
> >        if @entity == 'Patient'
> >          @list = Patient.where(patient_id: params[:query])
> >          if @list.empty?
> > -          @list = Patient.where("name LIKE :prefix", prefix: "%#{params[:query]}%")
> > +          @list = Patient.where("first_name LIKE :prefix or last_name LIKE :prefix", prefix: "%#{params[:query]}%")
>
> nit: capitalize the "or" please?
>
> I kinda like having the SQL commands/operators be in all caps as a
> convention.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/awong-dev/SCIInterim/pull/114/files#r33186580>.
>